grapher/controls/CollapsibleList.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import { CollapsibleList, MoreButton } from "./CollapsibleList"
+import { collapsibleListSampleItems } from "./CollapsibleList.sampleInput"
+
+export default {
+    title: "CollapsibleList",
+    component: CollapsibleList,
```

Comment:
Total sidenote, and not sure if it would even be helpful here.

Storybook also has the cool thing called "argTypes", when then let's you have controls for quickly testing various scenarios with your components. I've got some basic ones going for FacetChart. Honestly it's kind of a pain to use because it's weakly typed and hard to figure out what the options are, but when you get it going it's kind of neat.

Thought it might be useful to have a range to play around with size behavior.

Reply:
Very interesting! I'll look into it.